diff --git a/babel.js b/babel.js
index e00e1a6b28b539c49d0d1ba46e3d3d9069c91e18..ab55590a4288bbbb62d23034b2ccbfabb649155d 100644
--- a/babel.js
+++ b/babel.js
@@ -133626,6 +133626,7 @@
   exports.registerPreset = registerPreset;
   exports.registerPresets = registerPresets;
   exports.transform = transform;
+  exports.transformAsync = (code, options) => transformAsync(code, processOptions(options));
   exports.transformFromAst = transformFromAst;
   exports.transformScriptTags = transformScriptTags;
   exports.version = version;
