diff --git a/dist/-internal/debug-info.js b/dist/-internal/debug-info.js
index 45733a399fc957c2b01109ce9fb37172dfd7d721..ae723167fdbd88e189c47edb9d9a0a3c9d69e5d5 100644
--- a/dist/-internal/debug-info.js
+++ b/dist/-internal/debug-info.js
@@ -1,4 +1,3 @@
-import { _backburner } from '@ember/runloop';
 import { debugInfoHelpers } from './debug-info-helpers.js';
 import { getPendingWaiterState } from '@ember/test-waiters';
 
@@ -22,7 +21,7 @@ const SCHEDULED_AUTORUN = 'Scheduled autorun';
  * @returns {boolean} True if `getDebugInfo` is present in backburner, otherwise false.
  */
 function backburnerDebugInfoAvailable() {
-  return typeof _backburner.getDebugInfo === 'function';
+  return false;
 }
 
 /**
@@ -33,7 +32,7 @@ function backburnerDebugInfoAvailable() {
  * @returns {MaybeDebugInfo | null} Backburner debugInfo or, if the getDebugInfo method is not present, null
  */
 function getDebugInfo() {
-  return _backburner.DEBUG === true && backburnerDebugInfoAvailable() ? _backburner.getDebugInfo() : null;
+  return null;
 }
 
 /**
diff --git a/dist/setup-context-BSrEM03X.js b/dist/setup-context-BSrEM03X.js
index 0f688234691707c6ee3a1b0326efdd0edb1ea5cb..2a97d25f25ff8c28c099ab6190f679922059c798 100644
--- a/dist/setup-context-BSrEM03X.js
+++ b/dist/setup-context-BSrEM03X.js
@@ -1,4 +1,5 @@
-import { _backburner, run } from '@ember/runloop';
+import { run } from '@ember/runloop';
+import { isRenderPending as _emberIsRenderPending } from '@ember/renderer';
 import { set, setProperties, get, getProperties } from '@ember/object';
 import { setOwner } from '@ember/application';
 import buildOwner from './build-owner.js';
@@ -330,15 +331,15 @@ function _setupAJAXHooks() {
   @returns {Object} object with properties for each of the metrics used to determine settledness
 */
 function getSettledState() {
-  const hasPendingTimers = _backburner.hasTimers();
-  const hasRunLoop = Boolean(_backburner.currentInstance);
+  const hasPendingTimers = false;
+  const hasRunLoop = false;
   const hasPendingLegacyWaiters = checkWaiters();
   const hasPendingTestWaiters = hasPendingWaiters();
   const pendingRequestCount = pendingRequests();
   const hasPendingRequests = pendingRequestCount > 0;
   // TODO: Ideally we'd have a function in Ember itself that can synchronously identify whether
   // or not there are any pending render operations, but this will have to suffice for now
-  const isRenderPending = !!hasRunLoop;
+  const isRenderPending = _emberIsRenderPending();
   return {
     hasPendingTimers,
     hasRunLoop,
@@ -777,7 +778,6 @@ function setupContext(base, options = {}) {
   setContext(context);
   const testMetadata = getTestMetadata(context);
   testMetadata.setupTypes.push('setupContext');
-  _backburner.DEBUG = true;
   _prepareOnerror(context);
   return Promise.resolve().then(() => {
     const application = getApplication();
