diff --git a/dist/ember-template-compiler.js b/dist/ember-template-compiler.js
index 6b97705af339ca7eef12ed2a5b6ebf587c3bdc50..36a2e115d193fe82ef0802394664058abfb2ef99 100644
--- a/dist/ember-template-compiler.js
+++ b/dist/ember-template-compiler.js
@@ -18734,29 +18734,6 @@ var define, require;
                     default: bootstrap
           }, Symbol.toStringTag, { value: 'Module' });
 
-          // Globals mode template compiler
-          if (emberApp__namespace.default) {
-            let Application = emberApp__namespace.default;
-            let {
-              hasTemplate,
-              setTemplate
-            } = emberGlimmer__namespace;
-            let {
-              hasDOM
-            } = emberinternalsBrowserEnvironmentIndex;
-            Application.initializer({
-              name: 'domTemplates',
-              initialize() {
-                if (hasDOM) {
-                  bootstrap({
-                    context: document,
-                    hasTemplate,
-                    setTemplate
-                  });
-                }
-              }
-            });
-          }
 
           const emberTemplateCompilerLibSystemInitializer = /*#__PURE__*/Object.defineProperty({
                     __proto__: null
