diff --git a/lib/tasks/try-each.js b/lib/tasks/try-each.js
index 0967ac084811d315119cc059f619762d91876fed..96839eaf4c300d1769d117ad530f822576622459 100644
--- a/lib/tasks/try-each.js
+++ b/lib/tasks/try-each.js
@@ -140,7 +140,7 @@ module.exports = CoreObject.extend({
   },
 
   _defaultCommandArgs() {
-    return ['ember', 'test'];
+    return ['pnpm', 'turbo', 'run', 'test'];
   },
 
   _printResults(results) {